CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 ## Unreleased
 
+### Changed
+
+- Removed the `:width` and `:height` options from `Template.styles/1` and
+  `Template.source_and_options/1` as it turns out that Chrome does not pay attention to `@page`
+  dimensions and instead still sets the size of the produced PDF to US letter. Since I could not
+  figure out a way to set the PDF size besides using the `paperWidth` and `paperHeight` options,
+  I resided to moving to a `:size` option instead that accepts names like `:a4`, `:letter`, and
+  tuples of `{<width>, <height>}` in inches. These are then passed to `paperWidth` and
+  `paperHeight`.
```

Comment:
Possibly we could remove the `I` here as well and us a more neutral language. But since it is the changelogs, 🤷